generators/server/index.js

```diff
@@ -439,6 +439,9 @@ module.exports = class extends BaseBlueprintGenerator {
                 if (this.configOptions.clientFramework) {
                     this.clientFramework = this.configOptions.clientFramework;
                 }
+                if (this.configOptions.skipClient) {
+                    this.skipClient = this.configOptions.skipClient;
```

Comment:
Is there any use yet? Or is this just for future use, and to be sure the information is available in this subgenerator?

Reply:
Yes it's important as some files of the server are different if skipClient is set (pom.xml, README, ...)

Reply:
Ok, but this assignation is already done in `this.setupServerOptions(this)`: https://github.com/jhipster/generator-jhipster/blob/master/generators/server/index.js#L61

Reply:
It needs to be done again at `configuring` phase after the `prompting` phase. You can try to remove it and you'll see the server will be generated with front-end related code.

Reply:
Ok, I see. The `prompting` phase of the `client` subgen will indeed happen between the different phases of the `server` subgen.